tests/api/test_items_rest_views.py

```diff
@@ -726,7 +726,8 @@ def test_update_loan_pickup_location(
     )
     assert res.status_code == 200
     assert data.get('pickup_location_pid') == new_loc_pid
-    # Change loan state to 'ITEM_AT_DESK'
+    # Change loan state to 'ITEM_AT_DESK'.
+    # WARNING: Use loan from test_patron_checkouts_order (item3_lib_martigny)
```

Comment:
Is this a valid comment? because it is using the `item3_lib_martigny`
`# WARNING: Use loan from test_patron_checkouts_order (item3_lib_martigny)`

Reply:
Yes! This explain that we use an item from a previous test!